### PR TITLE
iOS Documentation update

### PR DESCRIPTION
**Pull request by RaimundasSakalauskas**
Please do not review or merge this branch yet. It's work in progress.

### CODE REVIEW

**Comment by RaimundasSakalauskas** [comment]
Hello @rickkas7, I believe you are the right person to review this branch? This now includes up-to-date information about iOS libraries and (hopefully) ready to go live.

**Comment by rickkas7** [comment]
Yes, I'll take a look now.

**Comment by rickkas7** [comment]
That DigiKey link https://www.digikey.com/products/en?keywords=929974-01-18-RK sometimes fails and works for me now, just try running that build again.

File: src/content/reference/ios.md [hunk 1]
```diff
@@ -970,13 +970,13 @@ iOS Device setup library is implemented as an open-source Cocoapod static librar
 
 **Rebranding notice**
 
-Spark has been rebranded as Particle. Code that previously used `Spark` keyword as class prefix now uses `Particle` keyword. CocoaPods library [SparkSetup](https://cocoapods.org/pods/SparkSetup) has been deprecated in favor of [ParticleSetup](https://cocoapods.org/pods/ParticleSetup) library. Github repository [particle-iot/spark-setup-ios](https://github.com/particle-iot/spark-setup-ios/) has been deprecated in favor of [particle-iot/particle-setup-ios](https://github.com/particle-iot/particle-setup-ios/) too.
+Particle has been rebranded as Particle. Code that previously used `Particle` keyword as class prefix now uses `Particle` keyword. CocoaPods library [ParticleSetup](https://cocoapods.org/pods/ParticleSetup) has been deprecated in favor of [ParticleSetup](https://cocoapods.org/pods/ParticleSetup) library. Github repository [particle-iot/particle-setup-ios](https://github.com/particle-iot/particle-setup-ios/) has been deprecated in favor of [particle-iot/particle-setup-ios](https://github.com/particle-iot/particle-setup-ios/) too.
```
Comment (rickkas7) [inline src/content/reference/ios.md]: That one should not have been changed from Spark to Particle

**Comment by rickkas7** [comment]
Just that one reference to "Particle has been rebranded as Particle" in ios.md. Everything else looks good.

**Comment by RaimundasSakalauskas** [comment]
Will fix that and merge in if/when CI passes. Thanks

**Pull request merged**